include/f_core/radio/c_lora.h

```diff
@@ -47,11 +47,18 @@ class CLora {
     int TransmitAsynchronous(const void* data, size_t len, k_poll_signal* signal);
 
     /**
-     * Receive data asynchronously (non-blocking)
+     * Enable receiving asynchronously. Should only be called once to start.
```

Comment:
minor nitpick about the comment - this comment makes it sound like you call this and forget but really it gets called multiple times if you temporarily disable with DisableAsynchronous and need to turn it back on

Reply:
Blasted the second sentence